File: api-quarkus/src/test/java/br/ufrn/musi/arquitetura/violacao/dominio/Contaminado.java
package br.ufrn.musi.arquitetura.violacao.dominio;

import jakarta.ws.rs.core.Response;

/** Fixture de ArquiteturaTest: um "domínio" que importa Jakarta REST. Não imite. */
public class Contaminado {
    public Response.Status status() {
        return Response.Status.OK;
    }
}

File: api-ktor/src/test/kotlin/br/ufrn/musi/arquitetura/violacao/dominio/Contaminado.kt
package br.ufrn.musi.arquitetura.violacao.dominio

import io.ktor.http.HttpStatusCode

/** Fixture de ArquiteturaTest: um "domínio" que importa Ktor. Não imite. */
class Contaminado {
    fun status(): HttpStatusCode = HttpStatusCode.OK
}
